### ISSUE TITLE
kublet error: "RegisterPlugin error"

### ISSUE THREAD

**Issue by floek** [kadalu/kadalu#429]
Hi,

I found the following error from Kubelet in my syslog:

```console
Feb  2 11:10:44 myhost kubelet[28179]: I0202 11:10:44.599282   28179 reconciler.go:156] operationExecutor.RegisterPlugin started for plugin at "/var/lib/kubelet/plugins_registry/kadalu/csi.sock" (plugin details: &{/var/lib/kubelet/plugins_registry/kadalu/csi.sock 2021-01-31 20:27:41.066650536 +0100 CET m=+1163.872556829 <nil> })
[...]
Feb  2 11:10:44 myhost kubelet[28179]: E0202 11:10:44.601476   28179 goroutinemap.go:150] Operation for "/var/lib/kubelet/plugins_registry/kadalu/csi.sock" failed. No retries permitted until 2021-02-02 11:12:46.601445505 +0100 CET m=+140669.407352949 (durationBeforeRetry 2m2s). Error: "RegisterPlugin error -- failed to get plugin info using RPC GetInfo at socket /var/lib/kubelet/plugins_registry/kadalu/csi.sock, err: rpc error: code = Unimplemented desc = Method
not found!"
```

Do I have to worry about this?

floek

**Comment by amarts** [comment]
This corresponds to one of the unimplemented methods in our CSI. Was there a blockVolume PV Claim attempted on kadalu storage class? 

About worrying about it, if things are working w.r.to storage and PV provisioning, I don't think its an issue. (Ignore my limited knowledge on k8s ecosystem in general, to many things there :-/)

**Comment by clambin** [comment]
getting this error too when running a single (external) storage and a single pvc:

```
apiVersion: kadalu-operator.storage/v1alpha1
kind: KadaluStorage
metadata:
  name: kadalu-test
  namespace: kadalu
spec:
  type: External
  details:
    gluster_host: 192.168.0.11
    gluster_volname: kadalu
---
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: pvc-kadalu-test2
  namespace: default
spec:
  storageClassName: kadalu.external.kadalu-test
  accessModes:
  - ReadWriteMany
  resources:
    requests:
      storage: 100Mi
```

**Comment by leelavg** [comment]
@clambin is it seen on arm platform and after this error appears any tasks (~consuming pvc) are getting effected?

**Comment by clambin** [comment]
yes.  running arm64 w/ Ubuntu 21.04.

**Comment by amarts** [comment]
@clambin all ARM64 issues are now fixed with 0.8.6 release. Can you please check that version? If the issue is found, please reopen the bug.
